tests/test_transforms/test_decomposition/test_change_points_based/test_change_points_models/test_base_change_points.py

```diff
@@ -5,8 +5,8 @@
 
 from etna.datasets import TSDataset
```

Comment:
Let's create a separate file for ruptures, here you should only check the methods of the base class and in ruptures check `get_change_points`